docs/RUN_PROD.md

```diff
@@ -0,0 +1,261 @@
+# Details of deploying `diracx` in production
+
+The aim of this documentation is to give pointers on how to install the `diracx-charts` longside an existing `DIRAC` installation.
+
+Effectively, this means that you will be using your existing databases (`MySQL`, `OpenSearch`), and just install the new dependencies of `diracx`.
+
+We go here with the assumption that you have a `kubernetes` cluster at hand. If you do no have one, see the [k3s example](../k3s/README.md).
+
+
+If your central infrastructure already provide the following services, by all mean, use them.
+
+More configuration options are available, please refer to the [values.yaml](../diracx/values.yaml)
+
+## Getting started
+
+You can generate a template for your installation using
+`dirac internal legacy generate-helm-values`
+
+This file still needs to be edited (look for `TODO`).
+The following sections give you more details.
+
+## IDP
+
+To authenticate a VO, you need an IDP that can work with the Oauth2 Authorization with PKCE.
+
+How you exactly configure it is idp dependant, but here are the basics:
+
+* Redirection URLs should be the following
+```
+https://<youdiracx.invalid>/api/auth/device/complete
+https://<youdiracx.invalid>/api/auth/authorize/complete
+```
+* The client should be public (no authentication)
+* The necessary scopes are `email`, `openid`, `profile`
+* The needed grant type is authorization flow
+
+
+You can for example use [CERN SSO](SSO.md) as an IdP.
+
+Note that you still need to have the users registered in `diracx` by filling the `CsSync` section in the CS.
```

Comment:
Do we actually explain that in details anywhere? If yes, may be we should add a link pointing to the `CsSync` section in the CS.